src/styles/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+  border: $border-size dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  @keyframes move {
+    from {
+      transform: rotate(0);
+      transform-origin: $default-transform-origin;
+    }
+
+    to {
+      transform: rotate(360deg);
+      transform-origin: $default-transform-origin;
+    }
+  }
+
+  &__minutes-hand {
+    height: $minutes-hand-height;
+    width: $minutes-hand-width;
+    background-color: #0700ff;
+    position: absolute;
+    left: center-position(($minutes-hand-width / 2));
+    top: center-position($minutes-hand-height);
+
+    @include animation($default-minutes-hand-duration, steps(60));
+  }
+
+  &__seconds-hand {
+    height: $seconds-hand-heigth;
```

Comment:
There is a typo in the variable name `$seconds-hand-heigth`. It should be `$seconds-hand-height` to maintain consistency and avoid errors.